docker/repo.sls

```diff
@@ -8,7 +8,8 @@
 {% set humanname_old = 'Old ' if docker.use_old_repo else '' %}
 
 {%- if grains['os_family']|lower in ('debian',) %}
-{% set url = 'https://apt.dockerproject.org/repo ' ~ grains["os"]|lower ~ '-' ~ grains["oscodename"] ~ ' main' if docker.use_old_repo else docker.repo.url_base ~ ' ' ~ docker.repo.version ~ ' stable' %}
+{% set url = '[arch=' ~ grains["osarch"] ~ '] https://apt.dockerproject.org/repo ' ~ grains["os"]|lower ~ '-' ~ grains["oscodename"] ~ ' main' if docker.use_old_repo else '[arch=' ~ grains["osarch"] ~ '] ' ~ doc
```

Comment:
Close this with `%}`.

Reply:
You were faster then I could do the walk :)

Reply:
thanks.  Travis failure are unrelated to this PR.